style.css

```diff
@@ -0,0 +1,284 @@
+body{
+	background-color: #EEEEEE;
+}
+
+.wrapper {
+	text-align: center;
+}
+.navbar {
+	max-height: 50px;
+	padding: 25px;
+	top: 0;
+	left: 0;
+	display: inline-block;
+    background-color: white;
+    width: 100%;
+    padding-top: 0;
+    position: absolute;
+    box-shadow: rgba(0, 0, 0, 0.0976562) 0px 1px 2px;
+	text-align: right
+}
+.content{
+	
+    width: 63%;
+    margin-left: 4%;
+	margin-top: 80px;
+
+}
+.video-container {
+	display: inline-block;
+}
+.sidebar{
+	max-width:468px;
+	display:inline-block;
+    width: 27%;
+    float: right;
+    top: 92px;
+    position: absolute;
+    right: 5%;
+	background-color: white;
+	box-shadow: rgba(0, 0, 0, 0.0976562) 0px 1px 2px;
+}
+.description{
+	text-align: left;
+    background-color: white;
+    box-shadow: rgba(0, 0, 0, 0.0976562) 0px 1px 2px;
+    padding: 12px;
+    margin-top: 15px;
+
+}
+.description span{
+    font-weight: bold;
+}
+.video{
+	
+    background-color: white;
+    box-shadow: rgba(0, 0, 0, 0.0976562) 0px 1px 2px;
+    padding: 12px;
+    margin-top: 12px;
+
+}
+.logo {
+	height: inherit;
+	margin-left:25px;
+	float: left;
+}
+.navbar-right{
+    position: relative;
+    top: 15px;
+    right: 75px;
+
+}
+.menu-button {
+	background-image:url('./img/buttons.png');
+	background-color: transparent;
+	width:16px;
+	height:16px;
+	display: inline-block;
+	background-position: -198px -862px;
+	margin-bottom: 23px;
+	position: relative;
+	top: 25px;
+	float: left;
+}
+.upload-button {
+	background-image:url('./img/buttons.png');
+	background-color: transparent;
+	width:24px;
+	height:24px;
+	display:inline-block;
+	background-position: -125px -314px;
+	background-repeat: no-repeat;
+    margin-left: 25px;
+	position: relative;
+	top: 10px;
+
+}
+.help-button {
+	background-image:url('./img/buttons.png');
+	background-color: transparent;
+	width:16px;
+	height:21px;
+	display:inline-block;
+	background-position: -147px -230px;
+	background-repeat: no-repeat;
+    margin-left: 25px;
+	position: relative;
+	top: 10px;
+}
+
+.history-button {
+	background-image:url('./img/buttons.png');
+	background-color: transparent;
+	width:16px;
+	height:21px;
+	display:inline-block;
+	background-position: -213px -605px;
+	background-repeat: no-repeat;
+    margin-left: 25px;
+	position: relative;
+	top: 10px;
+}
+.search-input{
+		width: 45%;
+}
+
+.search-input, .signin-button, .search-button{
+	
+    display: inline-block;
+    margin-left: 25px;
+	height: 30px;
+}
+.search-button{
+	margin-left: 0px;
+	position: relative;
+	top: 8px;
+}
+.search-button-icon {
```

Comment:
Lovely work with the stylesheet! :+1:. Try as much as you can to keep your rules looking uniform, you have spaces before the opening brace in some rules and no spaces in others.